<launch>

  <node
 name="motor_controller" pkg="motors" type="motor_controller" />


<param name="/serial_node/port" value="/dev/ttyACM0" />

<node
name="serial_node" pkg="rosserial_python" type="serial_node.py"/>

</launch>

